README.md

```diff
@@ -38,6 +37,62 @@ Otherwise, if the environment variables `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCE
 the operator will use them to interact with the AWS API. You can find the YAMLs
 for deploying the resources using the environment access keys at [deploy/iam-env-credentials](deploy/iam-env-credentials).
 
+## OLM installation
+
+At this stage, we use a custom `CatalogSource` specific for this operator:
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: aws-nlb-helper-operator-catalog
+  namespace: openshift-marketplace
+  # Installed openshift-marketplace to enable multi-namespace support
+  # - https://bugzilla.redhat.com/show_bug.cgi?id=1779080
+spec:
+  sourceType: grpc
+  image: quay.io/3scale/aws-nlb-helper-operator-catalog:latest
+  displayName: AWS NLB Helper Operator
+  updateStrategy:
+    registryPoll:
+      interval: 30m
+```
+
+And can be installed via `Subscription`:
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: aws-nlb-helper-operator
+  namespace: aws-nlb-helper
+spec:
+  channel: alpha
+  installPlanApproval: Automatic
+  name: aws-nlb-helper-operator
+  source: aws-nlb-helper-operator-catalog
+  sourceNamespace: openshift-marketplace
+  config:
+    env:
+      - name: WATCH_NAMESPACE
+        value: aws-nlb-helper,3scale-saas
```

Comment:
This deployment lacks the **OperatorGroup** resource (without it, a Subscription is not enough).

Actually, this `WATCH_NAMESPACE` envvar don't need to be specified on the **Subscription**, should be in the **OperatorGroup**

and then on the controller manager ned to be sued the `metadata.annotations['olm.targetNamespaces']`

Reply:
OG example added in https://github.com/3scale-ops/aws-nlb-helper-operator/pull/18/commits/20d56e60b3bbb604c2fc593bbca6653435a086dc